src/app/dashboard/page.tsx

```
@@ -1,51 +1,272 @@
+              ) : sortedCampaigns.length === 0 ? (
+                <div className="bg-white rounded-lg border border-[#e5e5e5] p-8 text-center text-gray-500">
+                  No campaigns yet.
+                </div>
+              ) : (
+                <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6">
+                  {sortedCampaigns.slice(0, 6).map((c, i) => (
+                    <CampaignCard
+                      campaign={c}
+                      rank={i + 1}
+                      imageUrl={imageUrlsByCampaignId[c.campaign_id] ?? null}
+                    />
```

ESLint

[error] Missing "key" prop for element in iterator

(react/jsx-key)